aws/cloudformation-template-ipsec

```diff
@@ -832,4 +884,4 @@
             "Value": "https://github.com/hwdsl2/setup-ipsec-vpn#next-steps"
         }
     }
-}
+}
```

Comment:
It is recommended to leave a newline at the end of the file (looks like it was removed).